nncf/onnx/hardware/fused_patterns.py

```diff
@@ -82,6 +82,25 @@ def create_swish_with_hard_sigmoid() -> GraphPattern:
     return pattern
 
 
+@ONNX_HW_FUSED_PATTERNS.register(HWFusedPatternNames.HSWISH_ACTIVATION)
```

Comment:
Could you please why you did not include the `Div` operation here? As I remember, the h-shish is
```math
hswish(x) = x \frac{ReLU6(x+3)}{6}
```

Reply:
You are right

Reply:
Updated

Reply:
I checked out that the pattern applied correctly and the quantization scheme is correct on the particular model.